contracts/upgradeable_contracts/BasicOmnibridge.sol

```diff
@@ -221,7 +221,7 @@ abstract contract BasicOmnibridge is
         onlyIfUpgradeabilityOwner
         validAddress(_receiver)
     {
-        require(isBridgedTokenDeployAcknowledged(_token));
+        require(isRegisteredAsNativeToken(_token));
```

Comment:
How does it work? If I have a token A on the chain N1 and bridge it to the chain N2 as a token A*. Then I have an imbalance on the chain N2 (the mediator balance of A* on N2 > the mediator balance of A on N1). Will the admins be able to fix this imbalance?

Reply:
It is not possible to achieve imbalance on the bridged token side, since the mediator does not have any tokens there. Instead, it mints and burns them when needed. There is no way of forcing and locking tokens on the mediator contract as well, since the token contract always requires a successful execution of the callback on the bridge.

Thus, fixMediatorBalance is only helpful for tokens that are native w. r. t. the current side of the bridge.

Reply:
Great! thanks